web/styles/app_variables.css

```diff
@@ -231,6 +231,7 @@
     /* Text colors */
     --color-text-default: hsl(0deg 0% 20%);
     --color-text-message-default: hsl(0deg 0% 15%);
+    --color-text-message-header-archived: hsl(0deg 0% 50%);
```

Comment:
How did you pick this color, and do we need a different color for the dark theme?